Lib/idlelib/idle_test/test_config.py

```diff
@@ -439,21 +439,21 @@ def test_get_extensions(self):
             ''')
         eq = self.assertEqual
         iGE = idleConf.GetExtensions
-        eq(iGE(shell_only=True), [])
-        eq(iGE(), ['ZzDummy'])
-        eq(iGE(editor_only=True), ['ZzDummy'])
-        eq(iGE(active_only=False), ['ZzDummy', 'DISABLE'])
-        eq(iGE(active_only=False, editor_only=True), ['ZzDummy', 'DISABLE'])
+        eq(iGE(shell_only=True), ['ParenClose'])
+        eq(iGE(), ['ZzDummy', 'ParenClose'])
+        eq(iGE(editor_only=True), ['ZzDummy', 'ParenClose'])
+        eq(iGE(active_only=False), ['ZzDummy', 'ParenClose', 'DISABLE'])
+        eq(iGE(active_only=False, editor_only=True), ['ZzDummy', 'ParenClose', 'DISABLE'])
```

Comment:
I believe that all these changes to test_config only apply to extensions, and are therefore not needed.